CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ This file documents all notable changes to the GEOS-Chem repository starting in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - TBD
+### Changed
+- Renamed `Emiss_Carbon_Gases` to `CO2_Production` in `carbon_gases_mod.F90`
+
+### Fixed
+- Added a fix to skip the call to KPP when only CO2 is defined in the carbon simulation
+- Added fix to turn on ship emissions for CO2 in the carbon simulation
+- Update `HEMCO_Config.rc` for carbon simulation to read data based on carbon species used
```

Comment:
Updated

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2510/commits/68604792ea1f4340f37d1d33bf1782c16e48e795